superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -590,7 +591,12 @@ const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
       case MenuKeys.ExploreChart:
         // eslint-disable-next-line no-unused-expressions
         props.logExploreChart?.(props.slice.slice_id);
-        window.open(props.exploreUrl);
+        if (domEvent.metaKey || domEvent.ctrlKey) {
```

Comment:
A few questions here:
1) Are we sure this works for mac and PC browsers as expected? Seems like it should, but ¯\\\_(ツ)_/¯ 
2) Do we need tests to prevent this regressing again? 
3) Is there a reason we can't just use an HTML link here? I feel like we had that solved at one point.
4) Is it not amazing that this particular link has been the subject of like 20 PRs and Issues?

Reply:
1. Yes it should
2. Let me see what kind of tests I can add here
3. We now enable dropdown navigation for accessibility by controlling onClick handlers
4. Tests should hopefully fix this

Reply:
I will add the unit tests in a follow-up PR to unblock this bug fix